### PR TITLE
Add manager to process properties

### PR DESCRIPTION
**Pull request by nolanpro**
Fixes http://tickets.pm4overflow.com/tickets/632

Allow setting a manager user when configuring a process

image

### CODE REVIEW

**Review by caleeli — CHANGES_REQUESTED** [review]
The implementation is almost perfect, just add the IDE helpers

**Review by caleeli — APPROVED** [review]

**Review by caleeli — CHANGES_REQUESTED** [review]
The manager_id is returning null from process version.
1. Create a process
2. Assign a manager to the process
3. Run the process
4. Try to get the manager_id from the process version. It returns **null**
```
>>> $request=\ProcessMaker\Models\ProcessRequest::find(15);
>>> $request->process->manager_id
=> 5
>>> $request->processVersion->manager_id
=> null
```

File: ProcessMaker/Models/Process.php [hunk 1]
```diff
@@ -1216,4 +1218,61 @@ public function validateBpmnDefinition($addWarnings = false, &$warning = [])
         }
         return true;
     }
+
```
Comment (caleeli) [inline ProcessMaker/Models/Process.php:1221]: Maybe you could move this section to ProcessTrait, so it could be added to Process and ProcessVersion

Comment (nolanpro) [reply ProcessMaker/Models/Process.php:1221]: I like the idea of the Trait, will try it out

File: resources/lang/en.json [hunk 2]
```diff
@@ -346,7 +346,7 @@
   "Files (API)": "Files (API)",
   "Files": "Files",
   "Filter Controls": "Filter Controls",
-  "Filter": "Filter",
+  "Filter": "Filter",G
```
[the remark this answers is not in the document]
Comment (nolanpro) [reply resources/lang/en.json:349]: Oops

**Review by caleeli — APPROVED** [review]

**Review by boliviacoca — APPROVED** [review]

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://develop-processmaker.ci.processmaker.net

**Pull request merged**